src/index.ts

```diff
@@ -42,6 +42,15 @@ export type LiveActivityConfig = {
   progressViewLabelColor?: string
   deepLinkUrl?: string
   timerType?: DynamicIslandTimerType
+  padding?: number
+  paddingHorizontal?: number
+  paddingVertical?: number
+  paddingTop?: number
+  paddingBottom?: number
+  paddingLeft?: number
+  paddingRight?: number
```

Comment:
```suggestion
  padding?: number | {
    horizontal?: number
    vertical?: number
    top?: number
    bottom?: number
    left?: number
    right?: number
  }
```
But wrap the object into a separate type